.rubocop.yml

```diff
@@ -0,0 +1,79 @@
+require: rubocop-rspec
+
+AllCops:
+  TargetRubyVersion: 2.2
```

Comment:
This is a gem, so in theory we don't know which ruby versions it'll run against. I guess we should put the earliest supported version here..? What do other gem authors do?

Reply:
I think we constrain the code to the earliest possible version we want to support, and prevent Rubocop from suggesting new features (like `dig`) that aren't in that version.

✅